dev/io.openliberty.microprofile.openapi.internal.common/resources/io/openliberty/microprofile/openapi/internal/resources/OpenAPI.nlsprops

```diff
@@ -73,6 +73,10 @@ OPENAPI_MERGE_PROBLEM_PATH_CLASH=The {0} path from the {1} module clashes with a
 OPENAPI_MERGE_PROBLEM_PATH_CLASH.explanation=The path is two modules. Both instances of the path cannot be included in the same OpenAPI document. This message is returned as part of CWWKO1662.
 OPENAPI_MERGE_PROBLEM_PATH_CLASH.useraction=Either change one of the conflicting modules or configure the mpOpenApi-2.0 feature to exclude one of them. This problem can be caused if servers are included in the OpenAPI document and the server URL does not end with the context root of the application (for example, because the application is accessed at a different URL through a reverse proxy).
 
+OPENAPI_MERGE_PROBLEM_EXTENSION_CLASH=The {0} extension key from the {1} module clashes with the same extension key from the {2} module. {1} will not be merged.
+OPENAPI_MERGE_PROBLEM_EXTENSION_CLASH.explanation=Two modules declare the same extension with different values at the top level of the OpenAPI document. Both instances of the extension cannot be included in the same OpenAPI document. This message is returned as part of CWWKO1662.
+OPENAPI_MERGE_PROBLEM_EXTENSION_CLASH.useraction=Either change one of the conflicting modules to remove the extension or configure the mpOpenApi-2.0 feature to exclude one of them.
```

Comment:
Suggested edit:

Either change one of the conflicting modules to remove the extension or configure the mpOpenApi-2.0 feature to exclude one of the modules.